src/shared/i18n/I18NFrench.cpp

```diff
@@ -18,441 +18,441 @@
-    { I18N_TNMT_RED_FLAG_DROPPED, "the red flag dropped" },
-    { I18N_TNMT_BLUE_FLAG_DROPPED, "the blue flag dropped" },
-    { I18N_TNMT_TEAM_RED_SCORED1, "team red scores" },
-    { I18N_TNMT_TEAM_BLUE_SCORED1, "team blue scores" },
-    { I18N_TNMT_TEAM_RED_SCORED2, "TEAM RED SCORES" },
-    { I18N_TNMT_TEAM_BLUE_SCORED2, "TEAM RED SCORES" },
-    { I18N_TNMT_SR_RECORD, "new record of ${1}: ${2}" },
+    { I18N_TNMT_SB_SR_TITLE, "COURSE" },
+    { I18N_TNMT_SB_LAPS, "TOURS" },
+    { I18N_TNMT_SB_BEST, "MEILLEUR" },
+    { I18N_TNMT_SB_LAST, "DERNIER" },
+    { I18N_TNMT_SR_LEAD, "meneur:" },
+    { I18N_TNMT_SR_LAP, "tour:" },
+    { I18N_TNMT_SR_BEST, "meilleur:" },
+    { I18N_TNMT_SR_LAST, "dernier:" },
+    { I18N_TNMT_SR_NA, "" },
+    { I18N_TNMT_PLAYER_JOINS, "${1} à rejoint la partie" },
+    { I18N_TNMT_PLAYER_FELL_OFF, "${1} est tombé d'un étage" },
+    { I18N_TNMT_PLAYER_SUICIDE, "${1} a choisi le suicide" },
+    { I18N_TNMT_COIN_RETURNED, "la pièce est de nouveau dans sa base" },
+    { I18N_TNMT_COIN_CATCHED, "${1} a attrapé la pièce" },
+    { I18N_TNMT_COIN_DROPPED, "la pièce a été lachée" },
+    { I18N_TNMT_RED_FLAG_RETURNED, "le drapeau rouge est retourné à sa base" },
+    { I18N_TNMT_BLUE_FLAG_RETURNED, "le drapeau bleu est retourné à sa base" },
+    { I18N_TNMT_RED_FLAG_CATCHED, "${1} a attrapé le drapeau rouge" },
+    { I18N_TNMT_BLUE_FLAG_CATCHED, "${1} a attrapé le drapeau bleu" },
+    { I18N_TNMT_RED_FLAG_SAVED, "le drapeau rouge a été sauvé par ${1}" },
+    { I18N_TNMT_BLUE_FLAG_SAVED, "le drapeau bleu a été sauvé par ${1}" },
+    { I18N_TNMT_RED_FLAG_DROPPED, "le drapeau rouge a été laché" },
+    { I18N_TNMT_BLUE_FLAG_DROPPED, "le drapeau bleu a été laché" },
+    { I18N_TNMT_TEAM_RED_SCORED1, "l'équipe rouge a marqué" },
+    { I18N_TNMT_TEAM_BLUE_SCORED1, "l'équipe bleue a marqué" },
+    { I18N_TNMT_TEAM_RED_SCORED2, "L'ÉQUIPE ROUGE MARQUE" },
+    { I18N_TNMT_TEAM_BLUE_SCORED2, "L'ÉQUIPE BLEUE MARQUE" },
+    { I18N_TNMT_SR_RECORD, "nouveau record de ${1}: ${2}" },
     { I18N_TNMT_SR_COMPLETE, "round completed" },
-    { I18N_TNMT_KILLED1, "${1} killed ${2}" },
-    { I18N_TNMT_KILLED2, "${1} killed himself" },
-    { I18N_TNMT_FROGGED1, "${1} frogged ${2}" },
-    { I18N_TNMT_FROGGED2, "${1} frogged himself" },
+    { I18N_TNMT_KILLED1, "${1} a tué ${2}" },
+    { I18N_TNMT_KILLED2, "${1} a mis fin à ses jours" },
+    { I18N_TNMT_FROGGED1, "${1} a grenouillé ${2}" },
+    { I18N_TNMT_FROGGED2, "${1} a courru sur sa propre grenouille" },
 
-    { I18N_TNMT_STATS_TEAM, "team stat" },
-    { I18N_TNMT_STATS_PLAYER, "player stat" },
-    { I18N_TNMT_STATS_OVER, "game over" },
-    { I18N_TNMT_STATS_END, "end of stats" },
-    { I18N_TNMT_STATS_MAP_CLOSED, "current map closed" },
+    { I18N_TNMT_STATS_TEAM, "statistiques de l'équipe" },
+    { I18N_TNMT_STATS_PLAYER, "statistiques du joueur" },
+    { I18N_TNMT_STATS_OVER, "fin de la partie" },
+    { I18N_TNMT_STATS_END, "statistiques de fin" },
```

Comment:
That's not really *the end of the stats* but *the stats you get at the end*. **IMO**